### PR TITLE
Why we don't provide webhosting services

### PR DESCRIPTION
**Pull request by therobot**
This article tries to tackle that question that I've seen both in groove and asked to me in person.

### CODE REVIEW

File: content/articles/why-we-dont-do-webhosting.markdown [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Why we don't offer web hosting services.
+excerpt: Welcome to DNSimple. This support page explains why we don't offer web hosting along our DNS hosting.
+categories:
+- DNSimple
+---
+
+# Why we don't offer web hosting services
+
+Traditionally DNS providers offered a basic web hosting solution along the DNS hosting service. It usually translated into giving several Megabytes of disk space served by an HTTP server and accessible with a FTP client to the user for HTML static sites.
```
Comment (jcaudle) [inline content/articles/why-we-dont-do-webhosting.markdown]: Would it be better to turn this sentence around and state that web hosting and DNS are usually bundled together? Or maybe point out the link that often exists between registrars and DNS?

File: content/articles/why-we-dont-do-webhosting.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- DNSimple
+---
+
+# Why we don't offer web hosting services
+
+Traditionally DNS providers offered a basic web hosting solution along the DNS hosting service. It usually translated into giving several Megabytes of disk space served by an HTTP server and accessible with a FTP client to the user for HTML static sites. 
+
+This can cause confusion to people without deep understanding of these technologies. This article explains why DNSimple does not offer web hosting. Some of the reasons are:
+
+ * Web hosting is not our core business. We do one thing and We do it very well, that is offering the best DNS hosting experience.
```
Comment (jcaudle) [inline content/articles/why-we-dont-do-webhosting.markdown]: What about

> We do one thing and we do it very well: offering the best DNS hosting experience.

File: content/articles/why-we-dont-do-webhosting.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+# Why we don't offer web hosting services
+
+Traditionally DNS providers offered a basic web hosting solution along the DNS hosting service. It usually translated into giving several Megabytes of disk space served by an HTTP server and accessible with a FTP client to the user for HTML static sites. 
+
+This can cause confusion to people without deep understanding of these technologies. This article explains why DNSimple does not offer web hosting. Some of the reasons are:
+
+ * Web hosting is not our core business. We do one thing and We do it very well, that is offering the best DNS hosting experience. 
+ * Today the web is way more diverse and complex. There is a miriad of technologies available for creating websites. These technologies use several pieces of software to work, database servers, web servers, caching services, load balancers, etc. Websites have heavier hardware requirements than some Megabytes of hard disk.
```
Comment (jcaudle) [inline content/articles/why-we-dont-do-webhosting.markdown]: s/miriad/myriad/

Also, this is a great point.

File: content/articles/why-we-dont-do-webhosting.markdown [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+# Why we don't offer web hosting services
+
+Traditionally DNS providers offered a basic web hosting solution along the DNS hosting service. It usually translated into giving several Megabytes of disk space served by an HTTP server and accessible with a FTP client to the user for HTML static sites. 
+
+This can cause confusion to people without deep understanding of these technologies. This article explains why DNSimple does not offer web hosting. Some of the reasons are:
+
+ * Web hosting is not our core business. We do one thing and We do it very well, that is offering the best DNS hosting experience. 
+ * Today the web is way more diverse and complex. There is a miriad of technologies available for creating websites. These technologies use several pieces of software to work, database servers, web servers, caching services, load balancers, etc. Websites have heavier hardware requirements than some Megabytes of hard disk. 
+ * The hosting service could only catter to a tiny fraction of our users that still make their HTML static websites.
```
Comment (jcaudle) [inline content/articles/why-we-dont-do-webhosting.markdown]: s/catter/cater/

**Comment by jcaudle** [comment]
This is a really valuable post. Thanks for writing it!

I wonder if we want to expand each of the bullet points more, but I think it'll be a good thing to :ship: for now!

**Comment by aeden** [comment]
:ship: ing it.

**Pull request merged**